portfolio/src/main/webapp/script.js

```diff
@@ -88,19 +91,20 @@ document.addEventListener('DOMContentLoaded', (event) => {
   form.addEventListener('submit', e => {
     e.preventDefault();
     fetch(scriptURL, { method: 'POST', body: new FormData(form), mode: "no-cors"})
-	  .then(response => respond("Thank you! Your rec was sent my way :)"))
+	  .then(response => respond("Thank you for sending :)"))
 	  .catch(error => respond("Uh oh, error: " + error.message))
+    
   });//the event occurred
 
   /**
  *Response to recommendation form
  */
   function respond(responseText) {
-    document.getElementById("form-response").innerHTML = responseText;
-    document.getElementById("form-response").style.visibility = "visible";
+    document.getElementById('form-response').innerHTML += responseText;
+    document.getElementById('form-response').style.visibility = "visible";
     setTimeout(() => {
```

Comment:
Since I don't see a demo for this PR, can you explain why we're doing a timeout to clear the form response?

Reply:
Sorry for forgetting- just updated the description. Demo: http://shershi-step-2020.appspot.com/ 

Once the second form succeeds in posting to Google Sheet, it will send back a message that will appear for 5 seconds before disappearing. So this timeOut just waits 5 secs before setting the message back to nothing.